.circleci/config.yml

```diff
@@ -54,3 +55,17 @@ jobs:
     docker:
       - image: php:7.2-alpine
 
+  test-code-quality:
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          name: 'Up and running php7'
+          command: 'docker-compose up'
+      - run:
+          name: 'Install dependencies'
```

Comment:
Curiosidade: este passo `Install dependencies` é realmente necessário? Pelo que entendi, o Psalm vai analisar o código do projeto de forma estática, então não precisa de `vendor` envolvida; e mesmo se for preciso fazer uma instalação para ter o Psalm, já tem um Dockerfile encarregado disso.

Reply:
@devdrops testei tirar esse step mas tive que manter pq o psalm requer que os autoloads e implementações (inclusive das dependências estejam instaladas =/.

Vou até ver no repo/issues se tem algo mencionando isso e validar com a galera se faz sentido ajustar isso.